test/tests/functional/pbs_pp369.py

```diff
@@ -36,10 +36,10 @@
 # "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
 # trademark licensing policies.
 
-from ptl.utils.pbs_testsuite import *
+from tests.functional import *
 
 
-class TestPp369(PBSTestSuite):
+class TestPp369(TestFunctional):
```

Comment:
@iestockdale  While working on this, I felt that it would be good to apply the naming convention guidelines to the existing tests. Hence, updated the same. I have updated the same in the ticket PP-461.